### PR TITLE
feat: add flatpickr bugfix rn

### PR DESCRIPTION
**Pull request by mritense**
-

### CODE REVIEW

File: release-notes/major9/09.23.0/valtimo-frontend-libraries.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ The following bugs were fixed:
   was defined. It will now only render a semicolon when both the `labelTranslationKey` and the `propertyPaths` are
   defined.
 
+* **Flatpickr inside Form.io form could not get locale**
+
+  When Flatpickr (date picker) was used inside a Form.io form, it could not properly determine the locale. The way the
```
Comment (marijnritense) [inline release-notes/major9/09.23.0/valtimo-frontend-libraries.md:56]: Can you rephrase this last sentence to:

The application language is passed to the Form.io component in a different way now, resolving the error.

Comment (mritense) [reply release-notes/major9/09.23.0/valtimo-frontend-libraries.md:56]: It's saying the same thing in a different way, both are grammatically correct and understandable to the reader.  On what basis are you making these comments?

**Review by theo-ritense — APPROVED** [review]
